mirgecom/eos.py

```diff
@@ -58,7 +60,7 @@ class EOSDependentVars:
     pressure: np.ndarray
 
 
-class GasEOS:
+class GasEOS(metaclass=ABCMeta):
```

Comment:
You can also just inherit from `ABC`. (Metaclasses might be confusing to folks.)

Reply:
Will address in #471.